docs/dev/log_integration.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Create a Log Integration
+kind: documentation
+aliases:
+- /logs/faq/partner_log_integration
+further_reading:
+- link: "/integrations/#cat-log-collection"
+  tag: "Documentation"
+  text: "See existing Datadog Log integrations"
+- link: "/logs/explorer/"
+  tag: "Documentation"
+  text: "Learn about the Log Explorer"
+- link: "/logs/log_configuration/pipelines/"
+  tag: "Documentation"
+  text: "Learn about log pipelines"
+---
+
+## Overview
+
+This page walks Technology Partners through creating a Datadog Log integration.
+
+## Log integrations
+
+Use the [Logs Ingestion HTTP endpoint][1] to send logs to Datadog. 
+
+## Development process
+
+### Guidelines
+
+When creating a log integration, consider the following best practices:
+
+: The `source` and `service` tags must be in lowercase. 
+
+
+Set the `source` tag to the integration name.
+: Datadog recommends that the `source` tag is set to `<integration_name>` and that the `service` tag is set to the name of the service that produces the telemetry. For example, the `service` tag can be used to differentiate logs by product line. </br></br> For cases where there aren't different services, set `service` to the same value as `source`. The `source` and `service` tags must be non-editable by the user because the tags are used to enable integration pipelines and dashboards. The tags can be set in the payload or through the query parameter, for example, `?ddsource=example&service=example`.
+
+The integration must support all Datadog sites.
+: The user must be able to choose between the different Datadog sites whenever applicable. See [Getting Started with Datadog Sites][2] for more information about site differences. </br></br> Your Datadog site endpoint is `http-intake.logs`.{{< region-param key="dd_site" code="true" >}}.
+
+You can attach custom tags while setting up the integration.
```

Comment:
This is supposed to tell the partner who is creating this integration that they should allow users to attach custom tags. The original line: "Allow the user to attach custom tags while setting up the integration."